ajax/libs/datatables.net-fixedheader-jqui/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "datatables.net-fixedheader-jqui",
```

Comment:
Maybe `DataTables-FixedHeader-jQueryUI` which is the same as the part of git repo name (also auto-update config) would be better and more meaningful?